src/screens/HomeScreen/component.js

```diff
@@ -124,12 +129,23 @@ const styles = StyleSheet.create({
     flex: 1,
     flexDirection: "row",
     flexWrap: "wrap",
+    justifyContent: "center",
     marginHorizontal: 12
   },
   tile: {
-    width: "50%",
-    paddingHorizontal: 4,
-    marginBottom: 8
+    width: "47%",
+    marginHorizontal: 4,
+    marginBottom: 12,
+    borderRadius: 3,
```

Comment:
The event image does not have the round corners at the top. This can be fixed by adding `overflow: "hidden"` here.

Reply:
Added to `EventTile` as it broke the shadow here